docs/CONTRIBUTING.md

```diff
@@ -16,30 +16,55 @@ please [see here](http://bit.ly/argo-wf-cmty-mtng).
 
 ## Contributor Meetings (twice monthly)
 
-A weekly opportunity for committers and maintainers of Workflows and Events to discuss their current work and
+An opportunity for committers and maintainers of Workflows and Events to discuss their current work and
 talk about what’s next. Feel free to join us! For Contributor Meeting information, minutes and recordings
 please [see here](https://bit.ly/argo-data-weekly).
```

Comment:
nice catch! I noticed this was no longer correct at some point too and forgot to fix it 😅 

while we're at it, could we get this paragraph to match the docs style guide too?

```suggestion
An opportunity for contributors and maintainers of Workflows and Events to discuss their current work and talk about what’s next. Feel free to join us!
See the [Contributor Meeting doc](https://bit.ly/argo-data-weekly) for minutes, recordings, and more information.
```